inlong-agent/agent-installer/src/main/java/org/apache/inlong/agent/installer/ModuleManager.java

```diff
@@ -81,8 +95,8 @@ public HttpManager getHttpManager(InstallerConfiguration conf) {
                 DEFAULT_AGENT_MANAGER_VIP_HTTP_PREFIX_PATH);
         int timeout = conf.getInt(AGENT_MANAGER_REQUEST_TIMEOUT,
                 DEFAULT_AGENT_MANAGER_REQUEST_TIMEOUT);
-        String secretId = conf.get(MANAGER_AUTH_SECRET_ID);
-        String secretKey = conf.get(MANAGER_AUTH_SECRET_KEY);
+        String secretId = conf.get(MANAGER_AUTH_SECRET_ID, "");
```

Comment:
This modification changes the behavior.
Without this pull request, this line will throw an exception if the `secretId` is not configured, but nothing happens with this pull request.